src/Vibrate.Plugin.Tizen/Vibrate.cs

```diff
@@ -0,0 +1,38 @@
+﻿using Plugin.Vibrate.Abstractions;
+using System;
+using Tizen.System;
+
+namespace Plugin.Vibrate
+{
+	/// <summary>
+	/// Vibration Implentation on Android 
+	/// </summary>
+	public class Vibrate : IVibrate
+	{
+		/// <summary>
+		/// Gets if device can vibrate
+		/// </summary>
+		public bool CanVibrate => true;
+
+		/// <summary>
+		/// Vibrate the phone for specified amount of time
+		/// </summary>
+		/// <param name="vibrateSpan">Time span to vibrate. 500ms is default if null</param>
+		public void Vibration(TimeSpan? vibrateSpan = null)
+		{
+			var milliseconds = vibrateSpan.HasValue ? vibrateSpan.Value.TotalMilliseconds : 500;
+			Vibrator vibrator = Vibrator.Vibrators[0];
```

Comment:
Is this guaranteed? Should we check first to make sure it has atleast one in it?